.github/CODEOWNERS

```diff
@@ -181,6 +181,7 @@ syndat_qctropcy.sh @JiayiPeng-NOAA
 tropcy_relocate.sh @JiayiPeng-NOAA 
 tropcy_relocate_extrkr.sh @JiayiPeng-NOAA 
 wave_*.sh @JessicaMeixner-NOAA @sbanihash
+regrid_gsiSfcIncr_to_tile.sh @ClaraDraper-NOAA
```

Comment:
```suggestion
ush/regrid_gsiSfcIncr_to_tile.sh @ClaraDraper-NOAA
```

Reply:
We need to add @ClaraDraper-NOAA to the NOAA-EMC/global-workflow-devs team w/ triage permissions.

Reply:
> 

@DavidHuber-NOAA This is not consistent with the rest of the ush files (which don't have ush/). Shall I update them all while I'm here?

Reply:
You are correct, my apologies.  Feel free to change this back.